core/src/com/geekbrains/td/BulletEmitter.java

```diff
@@ -27,11 +28,19 @@ public BulletEmitter(GameScreen gameScreen) {
         this.tmp = new Vector2(0, 0);
         this.templates = new HashMap<String, BulletTemplate>();
         this.loadTemplates();
+        hasCost = false;
```

Comment:
`private boolean hasCost;` is `false` by default and doesn't need an implicit initialization.